x/fungible/types/message_deploy_fungible_coin_zrc20.go

```diff
@@ -69,9 +69,6 @@ func (msg *MsgDeployFungibleCoinZRC20) ValidateBasic() error {
 	if msg.Decimals > 77 {
 		return cosmoserrors.Wrapf(sdkerrors.ErrInvalidRequest, "decimals must be less than 78")
 	}
-	if msg.LiquidityCap.IsNil() {
```

Comment:
Not sure we should remove this check? Seems still relevant to me
`liquidityCap` itself can be nil, then default would be set, but we should still prevent using the value inside `liquidityCap` to be nil

Reply:
I'm think `ValidateBasic()` gets called in cosmos side code so we wouldn't have the ability to set the default value before this gets called.

The default value logic lives in `x/fungible/keeper/evm.go` right now.

Reply:
Ah ok I understand what you're saying. Restored the inner nil check and test: https://github.com/zeta-chain/node/pull/3387/commits/b3e52cf358b2328fead04684715a7f1e19a78b15